shell/list/workload.vue

```diff
@@ -31,7 +34,7 @@ export default {
 
     this.loadHeathResources();
 
-    if ( this.allTypes ) {
+    if ( this.allTypes && !this.perfConfig.advancedWorker ) {
```

Comment:
Workloads is a strange use-case since it's not really a type, in this case though since the component is using ResourceFetch, whatever's here will override what's in the mixin so it has to be done here. Since it's also a bit of a one-off case as well, it makes sense to leave the custom logic in this component rather than add another branch of logic to the re-useable mixin.